standard/statements.md

```diff
@@ -172,18 +180,20 @@ Execution of an empty statement simply transfers control to the end point of the
 *Example*: An empty statement can be used when writing a `while` statement with a null body:
 > ```csharp
 > bool ProcessMessage() {...}
-> void ProcessMessages() {
+> void ProcessMessages()
+> {
 >     while (ProcessMessage())
->     ;
+>         ;
 > }
 > ```
 > Also, an empty statement can be used to declare a label just before the closing "`}`" of a block:
 > ```csharp
-> void F() {
->    ...
->    if (done) goto exit;
->    ...
->    exit: ;
+> void F()
+> {
+>     ...
+>     if (done) goto exit;
+>     ...
+>   exit: ;
```

Comment:
Do you mean to *not* put the goto on its own line inside braces?

Reply:
Nope, missed that one.

Reply:
Done - and moved the `;` to a line on its own too.